apps/dokploy/templates/sentry/index.ts

```diff
@@ -0,0 +1,77 @@
+import {
+	type DomainSchema,
+	type Schema,
+	type Template,
+	generateBase64,
+	generateRandomDomain,
+} from "../utils";
+
+export function generate(schema: Schema): Template {
+	const mainDomain = generateRandomDomain(schema);
+	const secretBase = generateBase64(64);
+	const totpKeyBase = generateBase64(32);
+
+	const domains: DomainSchema[] = [
+		{
+			host: mainDomain,
+			port: 8123,
+			serviceName: "clickhouse",
+		},
+		{
+			host: `${mainDomain}-smtp`,
+			port: 25,
+			serviceName: "smtp",
```

Comment:
This will not work because traefik.me-smtp is invalid hostname you need to generate another `generateRandomDomain`

btw why do you assign a domain to a email server?